examples/platforms/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -158,6 +158,18 @@
  */
 #define OPENTHREAD_CONFIG_HEAP_SIZE_NO_DTLS                     2048
 
+/**
+ * @def OPENTHREAD_CONFIG_INTERRUPT_CONTEXT_AES
+ *
+ * Define as 1 to enable AES usage in interrupt context.
+ *
+ * @note If this feature is enabled, a software AES under platform layer must be implemented when:
+ *       1. The platform's hardware AES only supports single AES engine;
+ *       2. The platform's hardware AES doesn't support interrupt context.
+ *
+ */
+#define OPENTHREAD_CONFIG_INTERRUPT_CONTEXT_AES 0
```

Comment:
What about moving this into `platform-config.h` and changing the name to something like `NRF_MBEDTLS_AES_ALT_INTERRUPT_CONTEXT`?